package com.rodrigodev.xgen.test.organization.packages.packagesForErrorsAreGeneratedFromErrorNames_specialCases.error_name1.error_name2.error_name3_7;

import com.rodrigodev.xgen.test.organization.packages.packagesForErrorsAreGeneratedFromErrorNames_specialCases.error_name1.error_name2.ErrorName2Exception;

/**
* Autogenerated by XGen on July 2, 2015.
*/
public class Error_name3_7Exception extends ErrorName2Exception {

    protected Error_name3_7Exception(String message) {
        super(message);
    }
}
